src/components/forms/selection-plan-form.js

```diff
@@ -1400,6 +1406,40 @@ class SelectionPlanForm extends React.Component {
                   />
                 </div>
               </div>
+              <div className="row form-group">
```

Comment:
enclose this section on a conditional asking for window.CFP_APP_BASE_URL presence
like
{ window.CFP_APP_BASE_URL &&